### PR TITLE
[Mobile] Implement Piano.

### PR DESCRIPTION
**Pull request by AchoWang**

### CODE REVIEW

File: Piano/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+### Features
+* Playing sound on key press.
+
+### Prerequisites
+* [Visual Studio](https://www.visualstudio.com/) - Buildtool, IDE
+* [Visual Studio Tools for Tizen](https://docs.tizen.org/application/vstools/install) - Visual Studio plugin for Tizen .NET application development
+
+### Author
+* Michał Kołodziejski
```
Comment (zg2nets) [inline Piano/README.md]: change author

**Review by zg2nets — CHANGES_REQUESTED** [review]

File: Piano/README.md (hunk 1 above)
Comment (AchoWang) [reply Piano/README.md]: Done,

**Review by zg2nets — APPROVED** [review]
lgtm

**Review by dongsug-song — APPROVED** [review]
approved

**Pull request merged**